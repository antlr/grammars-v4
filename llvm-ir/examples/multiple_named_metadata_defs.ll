!foo = !{!DIExpression(1)}
!foo = !{!DIExpression(2)}
