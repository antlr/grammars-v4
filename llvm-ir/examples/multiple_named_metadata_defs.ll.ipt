compilationUnit
 topLevelEntity
  namedMetadataDef
   !foo
   =
   !
   {
   metadataNode
    diExpression
     !DIExpression
     (
     diExpressionField
      1
     )
   }
 topLevelEntity
  namedMetadataDef
   !foo
   =
   !
   {
   metadataNode
    diExpression
     !DIExpression
     (
     diExpressionField
      2
     )
   }
 <EOF>
