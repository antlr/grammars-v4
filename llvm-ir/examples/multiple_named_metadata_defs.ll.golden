!foo = !{!DIExpression(1), !DIExpression(2)}
